install.sh

```diff
@@ -51,28 +53,31 @@ spec:
     registryPoll:
       interval: 10m0s
 EOD
+  fi
+fi
+
 #Apply subscription for subscribing to the requested channel of the service binding operator
 kubectl apply -f - << EOD
 ---
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
   name: service-binding-operator
-  namespace: operators
+  namespace: $OPERATOR_NAMESPACE
 spec:
   channel: $OPERATOR_CHANNEL
   installPlanApproval: Automatic
   name: service-binding-operator
-  source: sbo-operators
-  sourceNamespace: olm
+  source: $CATSRC_NAME
```

Comment:
what is the default behaviour when the script is invoked as

```
./install.sh
```

would it work at all?

Reply:
it does, now